client/engine/chainservice/eth_chainservice.go

```diff
@@ -61,11 +61,11 @@ const MAX_QUERY_BLOCK_RANGE = 2000
 const RESUB_INTERVAL = 2*time.Minute + 30*time.Second
 
 // NewEthChainService is a convenient wrapper around NewEthChainService, which provides a simpler API
```

Comment:
Out of scope, but
```suggestion
// NewEthChainService is a convenient wrapper around newEthChainService, which provides a simpler API
```
and also on L81.